release-notes/v0.6.0.md

```diff
@@ -0,0 +1,153 @@
+
+To enable parallel build, open `settings.json` and set the option `java.maxConcurrentBuilds` to a numeric value. The recommended value is the number of CPU cores on your machine.
+
+```json
+{
+    "java.maxConcurrentBuilds": 4 // on a 4 core machine
+}
+```
+
+### Extension Load Time Reduced
+
+The load time of Java extensions is reduced by adopting *webpack*. As you know, all VS Code extensions are written in JavaScript/TypeScript. Recently, we started to adopt webpack to generate the production packages, with code combined and minified. This dramatically reduced the extension load time.
+
+Please update the extensions to the newest version, and you'll get this improvement automatically.
+
+## Maven
+
+### Dependency Auto-Completion for `pom.xml` files
+
+Maven extension now provides code snippet to quickly add new dependencies to `pom.xml` files. You also get auto-completion when further completing the dependency info.
+
+![Dependency Auto-Completion](v0.6.0/pom-dependency-completion.gif)
+
+### Maven Plugin Goals
+
+Plugins and their goals are now listed in the explorer along with other Maven resources. You can execute the goals with a few clicks.
+
+![Maven Plugin Goals](v0.6.0/maven-plugin-goals.png)
+
+### Flat View & Hierarchical View
+
+Maven explorer now allows you to switch between flat view and hierarchical view, to your preference.
+
+## Debugging
+
+### Easy Launching for Multi-Main-Class Projects
+
+When you have multiple main classes in your workspace, you can have a special launch configuration to launch whatever is in the active editor. This can be handy when you use hot keys. And here's the new configuration:
+
+```json
+{
+    "type": "java",
+    "name": "Debug (Launch) - Current File",
+    "request": "launch",
+    "mainClass": "${file}" // whatever main class in the active editor
+}
+```
+
+![Multiple Main Class Easy Launch](v0.6.0/multiple.main.entry.gif)
+
+## Testing
+
+### Test Configuration Reorganized
+
+Test configurations are very useful in special test setups. Those configurations were originally stored in `launch.test.json`, which generated lots of confusions according to the user feedback.
+
+We listened, and as a result, we **deprecated** `launch.test.json`, and replaced it with regular VS Code settings. Now, the test configurations stay in `settings.json`, which can be global or at the workspace level. And they look like this:
+
+```json
+"java.test.config": [
+    {
+        "name": "myTestConfiguration",
+        "workingDirectory": "${workspaceFolder}",
+        "args": [ "-c", "com.test" ],
+        "vmargs": [ "-Xmx512M" ],
+        "env": { "key": "value" },
+    },
+    {
+        // Other configuration entry...
+    }
+]
+```
+
+For more details, visit [Test Runner Configurations](https://github.com/Microsoft/vscode-java-test/blob/master/runner-config.md).
+
+### Navigate to Source from Test Explorer
```

Comment:
My bad. It's not `from Test Explorer` but `from Test Report` 😰

I can provide a gif for you if you need.